HelpMyStreetFE/HelpMyStreetFE/Services/AddressService.cs

```diff
@@ -60,8 +60,7 @@ public async Task<List<PostCodeDetail>> GetPostcodeDetailsNearUser(User user)
             foreach (var postcode in postCodes.Content.Postcodes)
             {
                 var street = string.Concat(postcode.AddressDetails[0].AddressLine1.Where(c => !char.IsNumber(c)));
```

Comment:
Can we just remove this method GetPostcodeDetailsNearUser(User user)?  I presume it is only to support Registration Step 4.